### PR TITLE
Editorial pass

### PR DESCRIPTION
**Pull request by tfpauly**
Purely editorial pass of use-it-or-lose-it; mainly trying to fix up wordings that I found awkward or misleading on a detailed read.

### CODE REVIEW

File: draft-iab-use-it-or-lose-it.md [hunk 1]
```diff
@@ -65,15 +65,15 @@ informative:
 
 The ability to change protocols depends on exercising the extension and version
 negotiation mechanisms that support change.  Protocols that don't use these
-mechanisms can find that deploying changes can be difficult and costly.
+mechanisms can find it difficult and costly to deploy changes.
```
Comment (DavidSchinazi) [inline draft-iab-use-it-or-lose-it.md:68]: nit: if we're modifying this perhaps we fix the fact that the protocol isn't the one deploying changes?

File: draft-iab-use-it-or-lose-it.md [hunk 2]
```diff
@@ -99,12 +99,12 @@ applicable to protocol deployments that have less scale and diversity, or
 operate under different constraints.
 
 
-# Implementations of Protocols are Imperfect {#implementations}
+# Imperfect Implementations Limit Protocol Evolution {#implementations}
```
Comment (DavidSchinazi) [inline draft-iab-use-it-or-lose-it.md:102]: I think I prefer MT's phrasing: there is no such thing as a perfect implementation :)

**Review by DavidSchinazi — APPROVED** [review]
Nice!

File: draft-iab-use-it-or-lose-it.md (hunk 2 above)
Comment (tfpauly) [reply draft-iab-use-it-or-lose-it.md:102]: Yeah, my problem with the current title is that the previous title doesn't actually explain why this is a problem... just saying that "protocols are imperfect" didn't feel like a summary of the entire section 2.

Comment (DavidSchinazi) [reply draft-iab-use-it-or-lose-it.md:102]: It's not "protocols are imperfect", it's "implementations (of protocols) are imperfect"

**Review by martinthomson — COMMENTED** [review]
I share David's reservations about the title, but it isn't strong.

File: draft-iab-use-it-or-lose-it.md (hunk 2 above)
Comment (chris-wood) [reply draft-iab-use-it-or-lose-it.md:102]: Maybe just "Protocol Evolution Limitations" since all implementations are imperfect? 🤷

**Pull request merged**